crates/phactory/src/prpc_service.rs

```diff
@@ -799,4 +799,16 @@ impl<Platform: pal::Platform> PhactoryApi for RpcService<'_, Platform> {
         let echo_msg = request.echo_msg;
         Ok(pb::EchoMessage { echo_msg })
     }
+
+    fn derive_ident_sk(&mut self, request: pb::DeriveIdentSkRequest) -> RpcResult<pb::DeriveIdentSkResponse> {
+        // white list
+        let permitted_derivation_info: [&[u8]; 1] = [b"pnetwork"];
```

Comment:
I'd suggest to just derive a key based on some hard coded derivation path. So far the whitelist + customized derivation path approach doesn't look so necessary

Reply:
The derivation path can be something like PhalaI2PKey